examples/experimental/NumpyTensor.ipynb

```diff
@@ -0,0 +1,99 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Setting up Sandbox...\n",
+      "Done!\n"
+     ]
+    }
+   ],
+   "source": [
+    "import syft as sy\n",
+    "import torch as th\n",
+    "import numpy as np\n",
+    "\n",
+    "sy.create_sandbox(globals(), False, False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from syft.frameworks.torch.tensors.interpreters.numpy import NumpyTensor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = NumpyTensor(numpy_tensor=np.array([[1,2,3,4]])).wrap()\n",
```

Comment:
Do you still need to wrap with the create_numpy_tensor helper?

Reply:
Yes because that section isn't using create_numpy_tensor

Reply:
fixed